lib/OktaAuth.ts

```diff
@@ -357,6 +365,36 @@ class OktaAuth implements SigninAPI, SignoutAPI {
     return this.token.revoke(refreshToken);
   }
 
+  async getSignOutRedirectUrl(options: SignoutRedirectUrlOptions) {
+    let {
+      idToken,
+      postLogoutRedirectUri,
+      state,
+    } = options;
+    if (!idToken) {
+      idToken = (await this.tokenManager.getTokens()).idToken as IDToken;
```

Comment:
With [this change](https://github.com/okta/okta-auth-js/pull/689/files#diff-f18cf5684cf053b7ffbc2723f4983f1a26432ef9eade50cbfa00a74a7793dedbR259) there will be a method `getTokensSync` which returns the tokens synchronously. Would there be value in this method being sync?

Reply:
That would be ideal!